www/docs/client/setup.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: setup
+title: Set up a tRPC Client
```

Comment:
```suggestion
title: Setup a tRPC Client
```

Reply:
What do you think of this? 

https://writingexplained.org/setup-vs-set-up-difference#:~:text=Both%20words%20have%20different%20meanings,in%20order%20or%20arranging%20them.

Reply:
If it's conscious please ignore! :D

Reply:
Ha no worries, yes I believe this is grammatically correct, but English is subtle so I'm happy to be convinced otherwise 🙂